contents/handbook/small-teams/marketing/mission.mdx

```diff
@@ -1 +1 @@
-Increase the number of [ICP](/handbook/strategy/ideal-customer-persona) visitors to our website
+Make PostHog a hub for founders and product engineers to learn and collaborate
```

Comment:
I like the new definition! Much better than "hang out" 😆

Nit: I would remove "founder" from here

Reply:
hub is good, i like hub

founders i think is confusing without it being more explicitly in the icp that we have. we are increasingly shifting a bit  towards smaller companies since that's driving most (!) of our growth - i think this is perhaps a separate item to address maybe next q then can update here + there?